From 805756445dcaa0489df61432bf24c35090faac47 Mon Sep 17 00:00:00 2001
From: Ronak Shah <ronak.shah@xilinx.com>
Date: Tue, 1 Sep 2020 07:07:06 -0700
Subject: [PATCH 1549/1851] v4l: xilinx: multi-scaler: Added streaming
 capabilities to device

commit e65dac707e35698727d4ad8ba7538013addb6c01 from
https://github.com/Xilinx/linux-xlnx.git

This streaming capability to V4L2 device is required to support
streaming I/O method.

Signed-off-by: Ronak Shah <ronak.shah@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-multi-scaler.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-multi-scaler.c b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
index d40ee9d21e33..4d70586003cb 100644
--- a/drivers/media/platform/xilinx/xilinx-multi-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
@@ -2381,7 +2381,8 @@ static int xm2m_msc_probe(struct platform_device *pdev)
 		*vfd = xm2msc_videodev;
 		vfd->lock = &xm2msc->dev_mutex;
 		vfd->v4l2_dev = &xm2msc->v4l2_dev;
-		vfd->device_caps = V4L2_CAP_VIDEO_M2M_MPLANE;
+		vfd->device_caps = V4L2_CAP_VIDEO_M2M_MPLANE |
+				   V4L2_CAP_STREAMING;
 
 		ret = video_register_device(vfd, VFL_TYPE_GRABBER, chan);
 		if (ret) {
-- 
2.31.1

